index.js

```diff
@@ -1,5 +1,127 @@
+/* eslint-disable consistent-return */
+/* eslint-env node */
+
 'use strict';
 
+const mergeTrees = require('broccoli-merge-trees');
+const Rollup = require('broccoli-rollup');
+const rollupConfig = require('./config/rollup');
+const outputConfig = require('./config/output');
+const path = require('path');
+
 module.exports = {
-  name: require('./package').name
+	name: require('./package').name,
+
+	isDevelopingAddon: () => true,
+
+	config(env, baseConfig) {
+		const options = baseConfig[this.name] || {};
+
+		env = env || process.env.EMBER_ENV;
+		const isProductionEnv = Boolean(env.match('prod'));
+		const url = baseConfig.rootURL || '';
+
+		const defaults = {
+			entrypointFileName: 'module-imports.js',
+			outputFileName: 'bundle.js',
+			outputPath: 'assets',
+			minify: isProductionEnv,
+			modules: false,
+			entrypointPaths: [],
+			autoImport: true
+		};
+
+		this.options = Object.assign(defaults, options);
+		this.options.rootURL = url;
+
+		this._setOutputOptions();
+	},
+
+	_setOutputOptions() {
+		this._outputFilePaths = this.options.entrypointPaths.map((dirname) =>
+			path.basename(dirname, this.options.entrypointFileName)
+		);
+
+		const config = outputConfig(this.app.project.targets);
+
+		this._outputConfigs = [config.legacy];
+
+		if (this.options.modules) {
+			this._outputConfigs.push(config.modules);
+		}
+	},
+
+	/* istanbul ignore next */
```

Comment:
Why ignore? You can add a unit test to check that when calling this function it adds the `@webcomponents/webcomponentsjs` package.

Reply:
Test included in 024e844